api-gradle/src/main/kotlin/phraseapp/PhraseAppPlugin.kt

```diff
@@ -3,7 +3,17 @@ package phraseapp
 import org.gradle.api.Plugin
 import org.gradle.api.Project
 import phraseapp.internal.platforms.Flutter
+import phraseapp.tasks.ClearTranslationsTask
+import phraseapp.tasks.DownloadTask
+import phraseapp.tasks.DuplicateKeysCheckTask
+import phraseapp.tasks.PlaceHolderCheckTask
+import phraseapp.tasks.PluralsCheckTask
+import phraseapp.tasks.UploadTask
+import phraseapp.tasks.deprecated.CheckTask
 import java.io.File
+import phraseapp.tasks.deprecated.ClearTranslationsTask as DeprecatedClearTranslationsTask
+import phraseapp.tasks.deprecated.DownloadTask as DeprecatedDownloadTask
+import phraseapp.tasks.deprecated.UploadTask as DeprecatedUploadTask
```

Comment:
⚠️ This can be considered as a breaking change too. If a consumer is using directly a class task instead of the plugin access, the upgrade will generate an error in the access to these classes. Keep deprecated class at the root level and add new task in `tasks` package. We'll remove them in the next major release.